src/open_inwoner/js/components/card/ToggleHideSelection.js

```diff
@@ -0,0 +1,60 @@
+export class ToggleHideSelection {
```

Comment:
Currently the logic here makes it so that ALL cards get the hide-me class when expanded, but i am not sure why - the scope is very tangled.
Also we do not actually seem to need a toggle for the expand-button (so there would be no need for toggling data-attributes).

I can make all cards show with this script and remove the button like this, but it is not pretty:

```export class ToggleHideSelection {
  static selector = '.card--toggle-hide'

  constructor() {
    this.icon = document.querySelector('.expand-icon')
    this.button = document.getElementById('toggle-hide-elements')
    this.button.addEventListener('click', this.toggleHide.bind(this))
    this.button.innerHTML =
      this.icon.outerHTML + this.button.dataset.labelReveal

    // Hide all but the first three cards by default
    const allCards = document.querySelectorAll(ToggleHideSelection.selector)
    const cardsTail = Array.from(allCards).slice(3)

    if (cardsTail.length === 0) {
      this.button.classList.add('hide-me')
    } else {
      cardsTail.forEach((element) => {
        element.classList.add('hide-me')
      })
    }
  }

  toggleHide(event) {
    event.preventDefault()

    // Toggle the expand/collapse state of the button
    const isExpanded = this.button.getAttribute('aria-expanded') === 'false'
    this.button.setAttribute('aria-expanded', isExpanded ? 'true' : 'false')
    this.icon.textContent = isExpanded ? 'expand_more' : 'expand_less'
    this.button.innerHTML =
      this.icon.outerHTML +
      (isExpanded
        ? this.button.dataset.labelReveal
        : this.button.dataset.labelCollapse)

    if (isExpanded) {
      console.log(`Cards are expanded`)
      this.button.remove()
    }

    // Toggle 'hide-me' class on cards beyond the first three
    const allCards = document.querySelectorAll(ToggleHideSelection.selector)
    const cardsTail = Array.from(allCards).slice(3)

    cardsTail.forEach((element) => {
      element.classList.toggle('hide-me', !isExpanded)
    })
  }
}

document
  .querySelectorAll(ToggleHideSelection.selector)
  .forEach((toggleHide) => new ToggleHideSelection(toggleHide))
```